### DISCUSSION TITLE
Dependabot: Do grouped version updates work across multiple entries?

### DISCUSSION THREAD

**Discussion by Andre601** [community/community#59782] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have a Dependabot setup for a repository with multiple modules, which means that it has one entry for each module.

Now GitHub released [grouped version updates](https://github.blog/changelog/2023-06-30-grouped-version-updates-for-dependabot-public-beta/) as a public beta and it seems to be something I've been waiting for a while now (And that I even suggested myself in the past).

Tho, there is one question that was on my mind for a while, which I would appreciate to get an answer for: Does it work across multiple entries?
As mentioned does my repository have multiple modules, from which some often share the same sets of dependencies. This means that I'll get multiple PRs for the same dependency across the different modules.
Does that grouped version option work across multiple entries? As in, if I would make an entry with the same name on multiple entries, would Dependabot recognize them as being the same and group them together in one PR?

It would help me a lot if that's the case. If not, then I hope it becomes a future feature of dependabot.

**Comment by Andre601** [comment]
I initially selected the wrong Topic area and now edited it. Sorry about any possible inconvenience this may have caused.

**Comment by djrmarques** [comment]
Hello @Andre601! Did you end up figuring out how to do this? I am trying to achieve the same atm. EDIT: Seems like they released it here https://docs.github.com/en/code-security/dependabot/working-with-dependabot/configuring-multi-ecosystem-updates

**Reply by Andre601** [reply]
No. I moved the project in question to another site and use renovate there which provides the feature of updating multiple modules in one PR

**Comment by jsoref** [comment]
I think dependabot discussions are hosted in [dependabot/dependabot-core/discussions](https://github.com/dependabot/dependabot-core/discussions)
